### DISCUSSION TITLE
Automation duplicate option should keep the name of the original automation in stead of 'New automation'

### DISCUSSION THREAD

**Discussion by tozzke** [home-assistant/frontend#13662] [Automations / scripts / scenes] [answered]
Often when I'm fine tuning or adding functions to existing automations, I tend to open the automation, disable it, duplicate it and start adding triggers/conditions/actions. Just to see if it has the effect I hoped for :). If it didn't, I can revert back to the original/previous automation quite easily.
Pre 2022.9.0 it kept the original name of the automation with '(Duplicate)' added to the name. Editting the name was easy... adding the date in brackets e.g. and I was done.

In version 2022.9.0 the duplicate option has changed and when you try to save it, it's called 'New automation' by default which makes it kinda hard to give it the name of the original automation and adding the date/whatever.

I'd like to see it show the original name again of the automation when duplicating

**Comment by matthiasdebaat** [comment, marked as answer]
I agree, I see this issue is created for it https://github.com/home-assistant/frontend/issues/13667
